UPGRADE-3.3.md

```diff
@@ -127,6 +127,34 @@ Finder
 
  * The `ExceptionInterface` has been deprecated and will be removed in 4.0.
 
+Form
+------
+
+ * Using the "choices" option in ``CountryType``, ``CurrencyType``, ``LanguageType``,
```

Comment:
I think we need to reword this a bit to make clear that this is only deprecated as long as you do not set the `choice_loader` option to `null` (something like "using the `choices` option without overriding the `choice_loader` option is deprecated").